### PR TITLE
New package: librosa

### PR DESCRIPTION
**Pull request by FlorentBecker**

### CODE REVIEW

**Comment by FlorentBecker** [comment]
travis dies because the log is too long

**Comment by FRidh** [comment]
Could you update your commit name following the [guide](http://hydra.nixos.org/build/26805176/download/1/nixpkgs/manual.html#chap-submitting-changes). Thanks.

**Comment by FlorentBecker** [comment]
commit messages rewritten

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -4912,6 +4912,64 @@ let
...
+
+  librosa = buildPythonPackage rec {
+    name = "librosa-${version}";
+    version = "0.4.0";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/l/librosa/librosa-0.4.0.tar.gz";
+      md5 = "8149a70a5a5783c3cceeb69de3e07458";
+    };
+
+    buildInputs = with self; with pkgs;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: if there are no buildInputs this line can  be omitted.

**Comment by FlorentBecker** [comment]
I was not able to run the test suite (it complains that it can't find module "tester"):

**Pull request merged**